channelserver/src/server.rs

```diff
@@ -1,21 +1,24 @@
 //! `ChannelServer` is an actor. It maintains list of connection client session.
 //! And manages available channels. Peers send messages to other peers in same
-//! channel through `ChannelServer`.
-
+//! channels through `ChannelServer`.
 use std::collections::{hash_map::Entry, HashMap};
 use std::fmt;
 use std::time::Instant;
 
-use actix::prelude::{Actor, Context, Handler, Recipient};
+use actix::prelude::*;
```

Comment:
Let's be explicit here too.

Reply:
(shakes fist at cut and paste from examples)